apps/studio/components/interfaces/Settings/General/CustomDomainConfig/CustomDomainTableHeaders.tsx

```diff
@@ -0,0 +1,37 @@
+export type DNSTableHeaderProps = {
+  display: string | undefined
```

Comment:
Does this need to have `undefined` as an option? The `if` statement doesn't account for the `undefined` condition and where `<DNSTableHeaders />` is used has a string fallback anyway

Reply:
I'll see what I can do...

Reply:
Fixed. :-)